fullscreen/model/remove-first-manual.html

```diff
@@ -23,6 +23,9 @@
             assert_equals(document.fullscreenElement, last);
             assert_equals(event.target, last);
             first.remove();
+            // Both /first/ and /last/ were removed from the top layer and
+            // thus the fullscreen element synchronously becomes null.
+            assert_equals(document.fullscreenElement, null);
```

Comment:
@upsuper, I noticed this test is still wrong. If the fullscreen element is null, as it is in Firefox Nightly here, then no fullscreenchange event should be fired, because of "If doc’s fullscreen element is null, then resolve promise with undefined and terminate these steps."

So we need to change either the spec of Firefox in some way. In your implementation, what is it that causes the event to be fired?